.github/workflows/contracts.yml

```
@@ -34,6 +34,24 @@ jobs:
           FORKING_URL: ${{ secrets.MAINNET_ETH_HOSTNAME }}
         run: yarn test:system
 
+  contracts-deployment-test:
```

```suggestion
  contracts-deployment:
```
Could we drop the `-tests` from the name? I don't see any tests performed in this job.

---

It is a test run of the deployment scripts on local hardhat network instance. I know the name is confusing. Maybe I'll update it to `contracts-deployment-dry-run`?

---

Oh, I see. Im fine with your proposition of the change.